src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -1,40 +1,934 @@
 /*
  * File: z_select.c
  * Overlay: ovl_select
- * Description:
+ * Description: Debug map select
  */
 
 #include "z_select.h"
+#include "alloca.h"
+#include "overlays/gamestates/ovl_title/z_title.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_select/D_80802390.s")
+void Select_LoadTitle(SelectContext* this) {
+    {
+        GameState* state = &this->state;
+        state->running = false;
```

Comment:
```suggestion
        GameState* gameState = &this->gameState;
        gameState->running = false;
```
Because this is more internal, using just `state` might be fine. I think I still have a slight preference to staying consistent with using `gameState` instead of `state` throughout where applicable since `state` is still a little too general for me, but it's not a strong opinion and you can leave it if you'd like. This comment would apply through the whole file